book/src/csi-windows.md

```diff
@@ -0,0 +1,99 @@
+# CSI Windows Suppoort
+
+## Status
+
+Status | Min K8s Version | Min CSI proxy Version | Min Node Driver Registrar Version | latest artifacts
+--|--|--|--|--
+Beta | 1.19 | 0.2.0 | 1.3.0 | [binary](gs://k8s-staging-sig-storage/csi-proxy/csi-proxy-master.exe)
+Alpha | 1.18 | 0.1.0 | 1.3.0 |
+
+
+## Overview
+
+CSI drivers (e.g. AzureDisk, GCE PD, etc.) are recommended to be deployed as containers. CSI driver’s node plugin typically runs on every worker node in the cluster (as a DaemonSet). Node plugin containers need to run with elevated privileges to perform storage related operations. However, Windows was not supporting privileged containers (Note: privileged containers a.k.a Host process is introduced as alpha feature in Kubernetes 1.22 very recently). To solve this problem, [CSI Proxy](https://github.com/kubernetes-csi/csi-proxy) is a binary that runs on the Windows host and executes a set of privileged storage operations on Windows nodes on behalf of containers in a CSI Node plugin daemonset. This enables multiple CSI Node plugin authors to execute privileged storage operations on Windows nodes without having to ship a custom privileged operation proxy (until privileged operations get enabled for containers in Windows in the future).
```

Comment:
couple of nits: 
"CSI Node plugin authors to execute" => "CSI Node plugins to execute" 
Let's remove "(until privileged operations get enabled for containers in Windows in the future)."

Reply:
done